test/e2e/e2e_test.go

```diff
@@ -464,6 +472,124 @@ var _ = ginkgo.Describe("Kueue", func() {
+
+			gomega.Eventually(func(g gomega.Gomega) {
+				g.Expect(k8sClient.Get(ctx, lookupKey1, sampleJob)).Should(gomega.Succeed())
+				g.Expect(sampleJob.Spec.Suspend).To(gomega.Equal(ptr.To(false)))
+				g.Expect(k8sClient.Get(ctx, wlKey, wll)).Should(gomega.Succeed())
+				util.ExpectWorkloadsToBeAdmitted(ctx, k8sClient, wll)
+			}, util.Timeout, util.Interval).Should(gomega.Succeed())
+
+			ginkgo.By("Change the Active field to suspend the job and check the job remains suspended and the workload unadmitted")
+			// Changing Active to false
+			wll.Spec.Active = ptr.To(false)
+			gomega.Expect(k8sClient.Update(ctx, wll)).Should(gomega.Succeed())
+
+			ginkgo.By("checking a second job starts after first job is suspended")
+			sampleJob2 = (&testingjob.JobWrapper{Job: *sampleJob2}).Image("gcr.io/k8s-staging-perf-tests/sleep:v0.0.3", []string{"50s"}).Obj()
+			wll2 := &kueue.Workload{}
+
+			gomega.Expect(k8sClient.Create(ctx, sampleJob2)).Should(gomega.Succeed())
+			wlKey2 := types.NamespacedName{Name: workloadjob.GetWorkloadNameForJob(sampleJob2.Name), Namespace: sampleJob2.Namespace}
+
+			gomega.Eventually(func(g gomega.Gomega) {
+				g.Expect(k8sClient.Get(ctx, jobKey2, sampleJob2)).Should(gomega.Succeed())
+				g.Expect(sampleJob2.Spec.Suspend).To(gomega.Equal(ptr.To(false)))
+				g.Expect(k8sClient.Get(ctx, wlKey2, wll2)).Should(gomega.Succeed())
+				util.ExpectWorkloadsToBeAdmitted(ctx, k8sClient, wll2)
+			}, util.Timeout, util.Interval).Should(gomega.Succeed())
+
+			// Checking job stays suspended
+			ginkgo.By("checking job is suspended")
+			gomega.Eventually(func() *bool {
+				gomega.Expect(k8sClient.Get(ctx, types.NamespacedName{Name: sampleJob.Name, Namespace: sampleJob.Namespace}, createdJob)).
+					Should(gomega.Succeed())
+				return createdJob.Spec.Suspend
+			}, util.Timeout, util.Interval).Should(gomega.Equal(ptr.To(true)))
+
+			ginkgo.By("checking the first job and workload stay suspended and unadmitted")
+			gomega.Consistently(func(g gomega.Gomega) {
+				// Job should stay pending
+				g.Expect(k8sClient.Get(ctx, types.NamespacedName{Name: sampleJob.Name, Namespace: sampleJob.Namespace}, createdJob)).
+					Should(gomega.Succeed())
+				g.Expect(createdJob.Spec.Suspend).To(gomega.Equal(ptr.To(true)))
+				// Workload should get unadmitted
+				g.Expect(k8sClient.Get(ctx, wlKey, wll)).Should(gomega.Succeed())
+				util.ExpectWorkloadsToBePending(ctx, k8sClient, wll)
+				// Workload should stay pending
+				g.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(wll), wll)).Should(gomega.Succeed())
+				g.Expect(workload.HasQuotaReservation(wll)).Should(gomega.BeFalse())
+				// Should have Evicted condition
+				isEvicting := apimeta.IsStatusConditionTrue(wll.Status.Conditions, kueue.WorkloadEvicted)
+				gomega.Expect(isEvicting).Should(gomega.BeTrue())
+			}, util.Timeout, util.Interval).Should(gomega.Succeed())
```

Comment:
```suggestion
			}, util.ConsistendDuration, util.Interval).Should(gomega.Succeed())
```

this constant is just 3s, which should be enough.